demisto-sdk secrets: fix to ignore secrets by disable-secrets-detection (#828)
demisto/demisto-sdk: CHANGELOG.md

* Fixed an issue in the **validate** command where validation would fail due to a required version bump for packs which are not versioned.
* Will use env var `DEMISTO_VERIFY_SSL` to determine if to use a secure connection for commands interacting with the Server when `--insecure` is not passed. If working with a local Server without a trusted certificate, you can set env var `DEMISTO_VERIFY_SSL=no` to avoid using `--insecure` on each command.
* Unifier now adds a link to the integration documentation to the integration detailed description.

> * Fixed an issue where the **secrets** command don't ignore lines with the comment # disable-secrets-detection

Reviewer: ```suggestion
* Fixed an issue in the **secrets** command where ignore secrets were not skipped needed.
```
